packages/contract/README.md

```diff
@@ -148,6 +148,10 @@ This function creates a new instance of the contract abstraction representing th
 
 Creates an instance of the contract abstraction representing the contract at its deployed address. The deployed address is a special value given to @truffle/contract that, when set, saves the address internally so that the deployed address can be inferred from the given Ethereum network being used. This allows you to write code referring to a specific deployed contract without having to manage those addresses yourself. Like `at()`, `deployed()` is thenable, and will resolve to a contract abstraction instance representing the deployed contract after ensuring that code exists at that location and that that address exists on the network being used.
 
+#### `MyContract.transactionHash`
+
+This property accesses the hash of the transaction in which the contract was `deployed()`.
```

Comment:
I think it would make more sense here to just say "in which the contract was deployed"; the `deployed()` method isn't particularly relevant here (seeing as that's not what actually does the deploying).